htmd/queues/acecloudqueue.py

```diff
@@ -169,6 +167,18 @@ def _createJobScript(self, fname, workdir, runsh):
             f.write('bash run.sh')
         os.chmod(fname, 0o700)
 
+    def _hackRunScript(self, directory):
+        import fileinput
+
+        bak_suffix = '.acecloud.bak'
```

Comment:
> If the backup parameter is given (typically as backup='.<some extension>'), it specifies the extension for the backup file, and the backup file remains around; by default, the extension is '.bak' and it is deleted when the output file is closed.

Are you sure it recognizes that `.acecloud.bak` is not `.bak` and deletes it? I assume you checked but just in case I'm asking

Reply:
well, it does leave it behind. even when I was using just `.bak` it was leaving it behind. Do you think it's a bug and I should do something safer? I know there are several ways of accomplishing what I am doing here but this `fileinput` stuff seemed to be the easier (and the closest to a `sed`)

Reply:
Nah I guess it differentiates between default and user-passed value somehow? If it stays it's ok